runtime/manual/node/npm_specifiers.md

```diff
@@ -41,7 +41,8 @@ import chalk from "npm:chalk@5";
 ```
 
 Some packages do not though, but you can specify their types with a
-[`@deno-types`](../advanced/typescript/types.md) directive. For example, using a
+[`@deno-types`](./runtime/fundamentals/ts_support.md) directive. For example,
```

Comment:
Nit: Wondering if we should rename that file to `typescript.md` instead of `ts_support.md`. Might be better for SEO.